media/css/mozorg/home/home-promo.less

```diff
@@ -1042,24 +1042,6 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
-// ViewSource Conf
-.promo-large-landscape.viewsource {
-    background-color: #00b085;
-
-    h2 {
-        padding-top: 120px;
-        .at2x('/media/img/home/voices/promos/viewsource/logo.png', 280px, 55px);
```

Comment:
I thought about it but then, thought perhaps to keep it around for future use? There is gonna be a ViewSource again after all :)